helper/helper.go

```diff
@@ -67,6 +70,43 @@ func Ptr[T any](value T) *T {
 	return &value
 }
 
+func CustomEncoding(filePath string, data []byte) (io.Reader, string, error) {
```

Comment:
Maybe give this a more explicit name? Like `FileAndDataToMultipart` or something that explicitly says what the function does